communication/CMakeLists.txt

```diff
@@ -51,6 +51,11 @@ if(${BUILD_COMM_TCP_PLAIN} OR ${BUILD_COMM_TCP_TLS})
         target_include_directories(bftcommunication_shared PUBLIC ${OPENSSL_INCLUDE_DIR})
         target_link_libraries(bftcommunication_shared PUBLIC ${OPENSSL_LIBRARIES} diagnostics)
         target_compile_definitions(bftcommunication_shared PUBLIC USE_COMM_TLS_TCP)
+        if(LEAKCHECK OR THREADCHECK)
+            # Workaround for BC-8010 - SSL socket gets stuck on shutdown
```

Comment:
BC-8010 is not public. Maybe describe it in a short sentence instead?
Let's remove it from the commit message as well.